src/PhpPresentation/Writer/PowerPoint2007/ContentTypes.php

```diff
@@ -69,7 +69,7 @@ public function render()
             $oSlideMaster->setRelsIndex($idx + 1);
             $this->writeOverrideContentType($objWriter, '/ppt/slideMasters/slideMaster' . $oSlideMaster->getRelsIndex() . '.xml', 'application/vnd.openxmlformats-officedocument.presentationml.slideMaster+xml');
             $this->writeOverrideContentType($objWriter, '/ppt/theme/theme' . $oSlideMaster->getRelsIndex() . '.xml', 'application/vnd.openxmlformats-officedocument.theme+xml');
-            foreach ($oSlideMaster->getAllSlideLayouts() as &$oSlideLayout) {
```

Comment:
Note for me : If we remove the reference, how the layoutNr & layoutId are updated ?